iron-fit-behavior.html

```diff
@@ -63,23 +80,25 @@
       },
 
       /**
-       * The element that should be used to position the element.
+       * The element that should be used to position the element. If not set, it will
+       * default to the parent node.
+       * @type {!Element}
        */
       positionTarget: {
-        type: Node
+        type: Element
       },
 
       /**
        * The orientation against which to align the element horizontally
-       * relative to the `positionTarget`. Possible values are left, right, auto.
+       * relative to the `positionTarget`. Possible values are `left, right, auto`.
```

Comment:
Nit pick: technically the possible values should be expressed as string literals in this doc.